mentors/ruby/r.json

```
@@ -16,5 +16,13 @@
     "avatar_url": "http://rishiip.github.io/assets/images/dp.jpg",
     "bio": "I love Ruby for its simplicity and readability. I have been using it professionally for more than 3 years. I love mentoring ruby because it giving me ability to learn something new."
 
-  }
+  },
+  {
+  "github_username": "rahulkeerthi",
+  "name": "Rahul Keerthi",
+  "link_text": null,
+  "link_url": "https://www.rahulkeerthi.com/",
+  "avatar_url": null,
+  "bio": "Ruby is fantastic. It's the first programming language I learned and the most memorable. I've been teaching Ruby since 2020."
+}
```

```suggestion
    "github_username": "rahulkeerthi",
    "name": "Rahul Keerthi",
    "link_text": null,
    "link_url": "https://www.rahulkeerthi.com/",
    "avatar_url": null,
    "bio": "Ruby is fantastic. It's the first programming language I learned and the most memorable. I've been teaching Ruby since 2020."
  }
```
Just some minor indentation changes.